Misc/NEWS.d/next/Library/2017-09-05-10-30-48.bpo-31350.dXJ-7N.rst

```diff
@@ -0,0 +1 @@
+Micro-optimize _get_running_loop to become up to 10% faster.
```

Comment:
Please replace _get_running_loop with:

```
:func:`asyncio.get_event_loop`
```

Reply:
_get_running_loop is part of public API, btw :)

Reply:
"_" marks private functions, no?